### PR TITLE
Issue_1

### PR DESCRIPTION
**Pull request by sylvanamagued**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -31,4 +31,21 @@
   to {
     transform: rotate(360deg);
   }
+}*/
```
Comment (ahmadheshamabdelkader) [inline src/App.css:34]: What is the use of this css ?

File: src/App.js [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
+  
+    </div>
+  );
+}
+
+export default App;
+
+/*function handleMenuClick(e) {
   console.log("click", e);
 }
```
Comment (ahmadheshamabdelkader) [inline src/App.js:25]: Please do not comment what you think is not needed, delete unnecessary code lines

File: yarn.lock [hunk 3]
```diff
@@ -4448,6 +4448,11 @@ boolbase@^1.0.0, boolbase@~1.0.0:
   resolved "https://registry.yarnpkg.com/boolbase/-/boolbase-1.0.0.tgz#68dff5fbe60c51eb37725ea9e3ed310dcc1e776e"
   integrity sha1-aN/1++YMUes3cl6p4+0xDcwed24=
 
+bootstrap@^4.4.1:
```
Comment (ahmadheshamabdelkader) [inline yarn.lock:4451]: Remove bootstrap from lock file

File: src/Content.js [hunk 4]
```diff
@@ -1,83 +1,73 @@
 import React from "react";
 import { Layout} from 'antd';
 import { Row, Col } from 'antd';
+import axios from 'axios';
 const {Content} = Layout;
-
```
Comment (ahmadheshamabdelkader) [inline src/Content.js:5]: For every component please name the component as the file for simplicity

File: src/Content.js [hunk 5]
```diff
@@ -1,83 +1,73 @@
...
-      <Col span={8}> <img  style={{
-            resizeMode: "center",
-            height: 290,
-            width: 190}} alt="" src="https://data.whicdn.com/images/219123435/original.jpg"/>
-            
-            </Col>
-    </Row>
- 
-       </div>
+                <Content
```
Comment (ahmadheshamabdelkader) [inline src/Content.js:44]: Please install "Prettier" code formatter https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: src/SideBar.js [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
 import { UserOutlined, LaptopOutlined, NotificationOutlined } from '@ant-design/icons';
-import FetchRandomImage from './Content';
+import FetchRandomImage from './FetchRandomImage';
 const { SubMenu } = Menu;
 const { Sider } = Layout;
 
-function SideBar (){
-    return (
-       
+class SideBar extends React.Component{
```
Comment (ahmadheshamabdelkader) [inline src/SideBar.js:8]: Isn't function SideBar() {return } the same as class SideBar extends React.component{render()} ?

Comment (sylvanamagued) [reply src/SideBar.js:8]: oui

**Pull request merged**